.github/workflows/rust.yml

```diff
@@ -0,0 +1,24 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
```

Comment:
Is this newline intentional? It looks a bit weird.

Reply:
Removed